DBSetup/createDB.sql

```diff
@@ -1,31 +1,40 @@
 create database VA_Trial;
 \c va_trial
+
+create table Patient (
+PatientID int NOT NULL,
+Deceased bool NOT NULL,
+FirstName text NOT NULL, 
```

Comment:
FirstName, LastName, and Email should not be enforced.
```suggestion
FirstName text, 
```